docs/apache-airflow-providers-elasticsearch/logging/index.rst

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
Changing the filename may break existing links. We should add redirects. I think we just need to create an `docs/apache-airflow-providers-elasticsearch/redirects.txt` file with the proper mapping.

Reply:
How do I do that? Could you show me an example / maybe past PR that did this ?  Do we have it explained somewhere?

Reply:
Ah. Found it! Good catch!

Reply:
Here is example: 
https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-google/redirects.txt
https://github.com/apache/airflow/blob/main/docs/apache-airflow/redirects.txt

Reply:
Yep. Found it. Fixed already in the latest push (I also checked that the redirection works).